tests/deployment_utils.py

```diff
@@ -246,6 +253,14 @@ def universum_runner(execution_environment, runner_without_environment):
     yield runner_without_environment
 
 
+@pytest.fixture(params=(False, True))
+def universum_runner_nonci(execution_environment, runner_without_environment, request):
```

Comment:
The name of this fixture implies it is used only for 'nonci' mode, while, as I see in code further, it actually is used both in basic and nonci modes.